app/cdap/components/PipelineTriggers/store/PipelineTriggersActionCreator.ts

```diff
@@ -381,57 +408,33 @@ export function generateRuntimeMapping(mapping) {
 }
 
 /**
- * Method to transform the single trigger to OR group trigger.
+ * Method to validate the incoming trigger argument mapping.
  * @param mapping - the argument mappings of the pipeline trigger.
- * @param propertiesConfig - the properties config of the pipeline trigger.
  * @param triggersGroupRunArgsToAdd - the run arguments of the pipeline group trigger.
- * @param selectedNamespace - the selected namespace.
- * @param triggeringPipelineInfo - the triggering pipeline's info.
+ * @returns the validation result, null for valid result, string error for invalid result.
  */
-export function configureGroupTriggersProperties(
-  mapping: any,
-  propertiesConfig: any,
-  triggersGroupRunArgsToAdd: ITriggersGroupRunArgs,
-  selectedNamespace: string,
-  triggeringPipelineInfo: ITriggeringPipelineInfo
-) {
-  const triggersGroupRunArgs = { ...triggersGroupRunArgsToAdd };
-  mapping.forEach((map) => {
-    const keySplit = map.key.split(':');
+export function validateTriggerMappping(
```

Comment:
so this function returns null if the result is valid and it returns a string if it is invalid?

Reply:
hmm okay i guess it tells you what it returns in the js doc - can you move a summary of that explanation to the top part of the description as well?

Reply:
yep ur right, updated